model/godleyIcon.h

```diff
@@ -33,8 +33,22 @@
 namespace minsky
 {
   class GodleyTableEditor;
-
-  class GodleyIcon: public ItemT<GodleyIcon>
+  
+  // Godley variables broken out in a separate structure, as copying is non-default
+  struct GodleyVars
+  {
+  public:	      
+    GodleyVars() {}
+    virtual ~GodleyVars() {}
+    // copy operations not deleted to allow ItemT<GodleyIcon> to compile
+    GodleyVars(const GodleyVars& x) {}
+    GodleyVars& operator=(const GodleyVars&) {return *this;}
+
+    virtual void update() = 0;     
```

Comment:
I guess this doesn't need to be there now.

Reply:
Still needed, otherwise the wheels come off worse than before during a copy.

Reply:
There is no longer any reason for update() to be called in a GodleyVars context.

Reply:
I have managed to fix this.